selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -794,6 +794,19 @@ public void waitMarkerInPositionAndClick(MarkerLocator markerLocator, int positi
     seleniumWebDriverHelper.waitAndClick(By.xpath(format(markerLocator.get(), position)));
   }
 
+  /**
+   * Waits marker with specified {@code markerLocator} on the defined {@code line number} and move
+   * cursor to it
+   *
+   * @param markerLocator marker's type, defined in {@link MarkerLocator}
+   * @param position line's number, where marker is expected
+   */
+  public void waitMarkerInPositionAndMoveCursor(MarkerLocator markerLocator, int position) {
```

Comment:
`moveCursorToLineWithMarker(MarkerLocator markerLocator, int line)` would explain the propose of method better, IMHO